src/main/groovy/com/bmuschko/gradle/clover/InstrumentCodeAction.groovy

```diff
@@ -95,7 +95,13 @@ class InstrumentCodeAction implements Action<Task> {
                 }
 
                 getMethodContexts().each {
-                    ant.methodContext(name: it.name, regexp: it.regexp)
+                    ant.methodContext(
```

Comment:
How do you want to handle support of clover versions < 3.1.10?
See: https://confluence.atlassian.com/display/CLOVER/methodContext 

maxAggregatedComplexity and maxAggregatedStatements will not be supported.  Is there a nice way to sniff it out?